commit 624bdbcf3afe304ad043baca43d46bfda2746e78
Author: John Spray <jspray@redhat.com>
Date:   Wed Nov 9 19:52:03 2016 +0000

    mgr: remove unnecessary C_StdFunction
    
    FunctionContext works fine and already existed.
    
    Signed-off-by: John Spray <john.spray@redhat.com>

diff --git a/src/mgr/PyModules.cc b/src/mgr/PyModules.cc
index 35b6ed0687b..8599815363c 100644
--- a/src/mgr/PyModules.cc
+++ b/src/mgr/PyModules.cc
@@ -437,7 +437,7 @@ void PyModules::shutdown()
   // Signal modules to drop out of serve()
   for (auto i : modules) {
     auto module = i.second;
-    finisher.queue(new C_StdFunction([module](){
+    finisher.queue(new FunctionContext([module](int r){
       module->shutdown();
     }));
   }
@@ -469,7 +469,7 @@ void PyModules::notify_all(const std::string &notify_type,
     auto module = i.second;
     // Send all python calls down a Finisher to avoid blocking
     // C++ code, and avoid any potential lock cycles.
-    finisher.queue(new C_StdFunction([module, notify_type, notify_id](){
+    finisher.queue(new FunctionContext([module, notify_type, notify_id](int r){
       module->notify(notify_type, notify_id);
     }));
   }
